### PR TITLE
Fix backspace bug

### PR DESCRIPTION
**Pull request by DZGoldman**
Fixes https://github.com/natejenkins/socialApp/issues/2824
The iterator for merging containers after one is deleted can breach the root node and manipulate the DOM outside of squire

### CODE REVIEW

**Comment by dginev** [comment]
Can't figure out if you're a genius who can solve any issue in one line, or you just have some amazingly nice issues.

**Comment by dginev** [comment]
Looks like the right fix to me btw

**Comment by DZGoldman** [comment]
I vote for the former

**Comment by DZGoldman** [comment]
1. Do you want the new build to part of the commit's diff though?

File: build/squire-raw.js [hunk 1]
```diff
@@ -3584,15 +3584,16 @@ function Squire ( root, config ) {
...
+    // nodes.  To do this with the 'selectionchange' event is trickier since our current procedure triggers
+    // a selection change
     // if ( 'onselectionchange' in doc ) {
     //   this.addEventListener( 'selectionchange', this._updatePathOnEvent );
     // } else {
       this.addEventListener( 'keyup', this._updatePathOnEvent );
       this.addEventListener( 'mouseup', this._updatePathOnEvent );
       this.addEventListener( 'mouseup', function(){
+          console.info("MOUSEUP")
+          // debugger
```
Comment (DZGoldman) [inline build/squire-raw.js:3596]: Strange diff here, but this appears to just be some left-over change in Editor.js that was never added to the build.

**Pull request merged**